1_fetch.R

```diff
@@ -0,0 +1,41 @@
+source("1_fetch/src/find_nwis_sites.R")
+source("1_fetch/src/download_DO_data.R")
+
+p1_targets_list <- list(
+  # Load harmonized WQP data product
+  tar_target(
+    p1_wqp_data,
+    readRDS(file = "1_fetch/in/DRB.WQdata.rds")),
+  tar_target(
+    p1_DO_sites_nwis,
+    get_drb_sites(drb_huc8s,DO_pcodes,site_tp_select)),
+  tar_target(
+    p1_daily_DO_csv,
```

Comment:
I think it'd be more readable to break this out into it's own function (maybe `get_daily_do_sites` and it's own target).

Reply:
I agree that this step - to filter and identify the continuous NWIS sites for which there is no instantaneous (sub-daily) data reported but there are daily statistics - wasn't very clear/readable, especially given the original placement within the `tar_target()` call. To make `1_fetch.R` more readable, I've created a new target `p1_nwis_sites_daily` that represents just the daily sites, which gets called in `get_daily_nwis_data()` to download the data for each nwis "dv" site.